tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -1023,6 +1023,27 @@ public int getTotalChunkNum() {
     return chunkMetadataList;
   }
 
+  /**
+   * get ChunkMetaDatas of given path sorting by offset
+   *
+   * @param path         timeseries path
+   * @param sortByOffset whether to use offset to sort the ChunkMetadatas
+   * @return List of ChunkMetaData
+   */
+  public List<ChunkMetadata> getChunkMetadataList(Path path, boolean sortByOffset)
```

Comment:
I've no idea whether the `sortByOffset` parameter is necessary, because as far as I known, the ChunkMetadataList has been sorted already in the disk.

Reply:
done